CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Next Version
 
 - Derived 'Copy' for 'KeyEvent'
+- Refactor the style module [issue](https://github.com/crossterm-rs/crossterm/issues/286)
+  - Renamed ObjectStyle to ContentStyle. Now full names are used for methods.
```

Comment:
Please, don't use past tense, just `Rename`. Also please use `backticks` for types, so, it's nicely formatted on the GitHub release page.

Reply:
Fixed